lib/column/fixed_string.go

```diff
@@ -178,7 +178,11 @@ func (col *FixedString) Append(v any) (nulls []uint8, err error) {
 			return nil, err
 		}
 
-		nulls = make([]uint8, len(data)/col.col.Size)
+		if col.col.Size == 0 {
```

Comment:
can we do something like following?
```
size := 0
if col.col.Size != 0 {
    size = len(data)/col.col.Size
}
nulls = make([]uint8, size)
```

Rationale being, this makes it more readable IMHO. Some additional advantages like
1. Less duplication with `make()`
2. Bit simpler control flow
3. Single place to calculate the value of `size`.

what do you think?